### PR TITLE
add format and prettier scripts

### PR DESCRIPTION
**Pull request by jikim-msft**
### Description

Board: https://dev.azure.com/fluidframework/internal/_workitems/edit/2118/

This PR allows running prettier in each sub-directory of release group and packages. Instead of adding the script to the root directory, developers can run `npm format` in any directory they are currently working on which will fix entire files accordingly to the prettier configuration.

There should be a separate PR to run prettier in Tools directory.

### CODE REVIEW

**Review by Josmithr — CHANGES_REQUESTED** [review]
We should also add prettier config files to each of the packages, otherwise the resulting formatting won't match our standards when run.

**Review by Josmithr — APPROVED** [review]
I actually think it would be okay to leave `useTabs` as true for these, since they weren't using prettier before, so there is no existing behavior we're trying to preserve. But either way is fine for now; the most important thing is getting the dependencies, configs, and scripts in place.

**Comment by jikim-msft** [comment]
> I actually think it would be okay to leave `useTabs` as true for these, since they weren't using prettier before, so there is no existing behavior we're trying to preserve. But either way is fine for now; the most important thing is getting the dependencies, configs, and scripts in place.

I was also thinking about setting `useTabs` to our global configuration, but also wanted to keep the `prettier.config` in similar fashion as others. I think removing that line makes more sense, I will remove the `useTabs` for tools directory.

File: tools/bundle-size-tools/prettier.config.cjs [hunk 1]
```diff
@@ -1,9 +0,0 @@
-/*!
- * Copyright (c) Microsoft Corporation and contributors. All rights reserved.
- * Licensed under the MIT License.
- */
-
-module.exports = {
```
Comment (Josmithr) [inline tools/bundle-size-tools/prettier.config.cjs]: I don't think you meant to revert the whole file here 🙂

**Review by Josmithr — APPROVED** [review]

File: tools/bundle-size-tools/prettier.config.cjs (hunk 1 above)
Comment (Josmithr) [reply tools/bundle-size-tools/prettier.config.cjs]: Never mind. Per our conversation on Teams, this package no longer exists (just a lingering directory with an npmignore file). Looks good!

**Pull request merged**